### DISCUSSION TITLE
Cloning Issue

### DISCUSSION THREAD

**Discussion by Fred-Goddy** [community/community#31435] [Other Feature Feedback, Questions, & Ideas]
After cloning with personal access token why is GitHub still requesting for password and what do I do?

**Comment by airtower-luna** [comment]
Short answer: Enter the token again. :slightly_smiling_face:

When you enter the token during clone, it is used to authorize access while cloning. It doesn't get saved anywhere. If you don't want to have to enter the token every time, check [Git Tools - Credential Storage](https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage). The `cache` method plus a password manager to store the token in the long term is usually a good option.

**Reply by Tecnophille** [reply]
Since a PAT can be used in place of a password when performing Git operations over HTTPS with Git on the command line or the API, you can use https://git-scm.com/docs/gitcredentials to cache it securely.
